mythril/analysis/modules/multiple_sends.py

```diff
@@ -62,51 +60,41 @@ def _analyze_state(state: GlobalState):
         list(state.get_annotations(MultipleSendsAnnotation)),
     )
     if len(annotations) == 0:
-        log.debug("Creating annotation for state")
         state.annotate(MultipleSendsAnnotation())
         annotations = cast(
             List[MultipleSendsAnnotation],
             list(state.get_annotations(MultipleSendsAnnotation)),
         )
 
-    calls = annotations[0].calls
+    call_offsets = annotations[0].call_offsets
 
     if instruction["opcode"] in ["CALL", "DELEGATECALL", "STATICCALL", "CALLCODE"]:
-        call = get_call_from_state(state)
-        if call:
-            calls += [call]
+        call_offsets.append(state.get_current_instruction()["address"])
 
     else:  # RETURN or STOP
-        if len(calls) > 1:
+        if len(call_offsets) > 1:
```

Comment:
I think we can skip this line in python. In python if `a = []` then `a[1:]` is also `[]`

Reply:
Wow, you're right :)